### ISSUE TITLE
Port monitoring doesn't seem to work

### ISSUE THREAD

**Issue by Tiwas** [AdyRock/homey-netscan#4]
I was hoping to monitor my internal web server with this. IP monitoring seems to work fine, but not port. I can access the TCP port (8000) by web browser and telnet, but the app claims it's down.

Is this a known problem? Any workaround?

**Comment by AdyRock** [comment]
Could you go to https://tools.developer.homey.app/tools/app-settings and select the NetScan app. Then make sure the Diagnostics log is set to Basic Information. Once the app has tried to detect the web server port, send the log so I can see what response it gets back.

**Comment by Tiwas** [comment]
Sent. 
![image](https://user-images.githubusercontent.com/17053402/156742343-12e99a97-019f-477a-9985-81612bb272ae.png)

**Comment by Tiwas** [comment]
Am I reading the logs correctly? It says, as far as I can see, in the logs that it's "still online", but the Homey app tells me it's not.

**Comment by AdyRock** [comment]
Ah, you are not the first one to be confused by the alarm name :-)
The "Device Offline" is the name of the alarm and the "No" is the status, so it is saying the device is not offline. The alarm is active if the device goes offline and the "No" changes to "Yes". Unfortunately Homey doesn't allow the status text to be changed from No / Yes to anything else. Ideally I would want to make the tile "Status" and the state text to be "Online / Offline" but that is not possible.
The app works this way from historical reasons as I took over the app from another developer that didn't want to maintain it any more and I rewrote it for SDK3 but wanted it to be compatible for previous users.
